test/helpers.js

```diff
@@ -5,9 +5,8 @@ const StripAnsi = require('strip-ansi');
 const Serverless = require('serverless');
 const ServerlessConfigFile = require('serverless/lib/utils/getServerlessConfigFile');
 const Offline = require('serverless-offline');
-const Somever = require('@hapi/somever');
 
-exports.Hapi = Somever.match(process.version, '>=12') ? require('@hapi/hapi-20') : require('@hapi/hapi');
+exports.Hapi = require('@hapi/hapi');
```

Comment:
This feels so good to get rid of <3

Reply:
I kept it in the helpers in case it's needed in the future

Reply:
I think that's fine 👍 I am just excited to get rid of this node version matching, etc. inside the test suite as you've done here :)